.github/workflows/gradle.yml

```diff
@@ -0,0 +1,12 @@
+name: Gradle CI
```

Comment:
@Sylux6
Sorry for my ignorance, but what is this file for?

It looks that every push triggers CI test now.
https://github.com/KenjiOhtsuka/harmonica/commits/develop
And, CircleCI configuration page says "By default, we will build all the commits for this project."
![image](https://user-images.githubusercontent.com/1572810/74136456-68962400-4c31-11ea-9262-0ea8e3ba2636.png)

Reply:
This is for the CI in GitHub, in the repo itself that you can find for yours [here](https://github.com/KenjiOhtsuka/harmonica/actions/new)

https://help.github.com/en/actions/automating-your-workflow-with-github-actions

Reply:
It's up to you to keep this or not though